measurement/signals/handlers.py

```diff
@@ -0,0 +1,40 @@
+from django.contrib.auth import get_user_model
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from guardian.shortcuts import assign_perm
+
+from measurement.models import MeasurementBase
+
+User = get_user_model()
+
+
+@receiver(post_save, sender=MeasurementBase)
```

Comment:
`MeasurementBase` is an abstract model, so it does not have a table in the DB and therefore it will never be sending any signal. It should be `Measurement` - and possibly `Report`. I'm not sure if it can be applied to both by providing a list to `sender`.